README.md

```diff
@@ -70,7 +81,7 @@ steps:
 
 See also the [example repo](https://github.com/erikmd/docker-coq-github-action-demo).
 
-### Inputs
+### Inputs to the action.yaml file
```

Comment:
Hmm, I understand what you mean, but as there isn't any `workflow.yml` file in this repo, I suppose a user would base it on `action.yaml` given that slighty above (under usage) it says "see `action.yml`".

This is what I see on the Github action marketplace
```
- name: Docker-Coq Action
  # You may pin to the exact commit or the version.
  # uses: coq-community/docker-coq-action@5ede7d711a7eae792579832cca39133bbc9d2644
  uses: coq-community/docker-coq-action@v1.2.2
  with:
    # The path of the .opam file (or a directory), relative to the repo root.
    opam_file: # optional, default is .
    # "8.X", "latest", or "dev".
    coq_version: # optional, default is latest
    # minimal, 4.07-flambda, or 4.09-flambda
    ocaml_version: # optional, default is minimal
    # Customizable script run before "install", empty by default.
    before_install: # optional, default is startGroup "Print opam config"
  opam config list; opam repo list; opam list
endGroup

    # Customizable script to install the opam dependencies.
    install: # optional, default is startGroup "Install dependencies"
  opam pin add -n -y -k path $PACKAGE $WORKDIR
  opam update -y
  opam install -y -j 2 $PACKAGE --deps-only
endGroup

    # Customizable script run after "install".
    after_install: # optional, default is startGroup "List installed packages"
  opam list
endGroup

    # Customizable script run before the opam build, empty by default.
    before_script: # optional
    # Customizable script run to install the opam package(s).
    script: # optional, default is startGroup "Build"
  opam install -y -v -j 2 $PACKAGE
  opam list
endGroup

    # Customizable script run after the opam build, empty by default.
    after_script: # optional
    # Customizable script run to uninstall the opam package(s).
    uninstall: # optional, default is startGroup "Uninstallation test"
  opam remove $PACKAGE
endGroup

    # The main script run in the container; may be overridden.
    custom_script: # optional, default is {{before_install}}
{{install}}
{{after_install}}
{{before_script}}
{{script}}
{{after_script}}
{{uninstall}}

    # The name of the Docker image to pull; unset by default.
    custom_image: # optional
    # Space-separated list of env variables to export to the custom_script.
    export: # optional
```

Reply:
what is the URL where you see this text as is?

because AFAIK, the standard GitHub marketplace URL is https://github.com/marketplace/actions/docker-coq-action and just renders the README.md in the first place. And the fact it mentions the `actions.yml` file was only a choice by mine but, admittedly is not very user-friendly (as I recall that `actions.yml` is a priori not required to be parsed by end-users, and definitely not intended to be edited by them): the README.md documentation should just be self-contained.

So I keep my suggestion of not writing `### Inputs to the action.yaml file` nor `### Inputs to the action.yml` but just `### Inputs to the docker-coq-action` or merely `### Inputs` if you prefer.

Reply:
BTW, do you agree of adding somewhere the text I had suggested [in my main comment](https://github.com/coq-community/docker-coq-action/pull/56#pullrequestreview-652829775)? Namely:

add some helpful text like this somewhere (e.g. just after `## Example`?):

> Using a [GitHub Action](https://docs.github.com/en/actions) in your GitHub repository amounts to committing a file `.github/workflows/your-workflow-name.yml`, e.g., `.github/workflows/build.yml`, containing (among others), a snippet such as:
> ```
> runs-on: ubuntu-latest  # container actions require GNU/Linux
> […]
>       ocaml_version: ${{ matrix.ocaml_version }}
> ```
> Each field can be customized, see [below](#inputs-to-the-docker-coq-action) for the documentation of those specific to the docker-coq-action, or the GitHub Actions official documentation for the [standard fields involved in workflows](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions).

Reply:
> BTW, do you agree of adding somewhere the text I had suggested [in my main comment](https://github.com/coq-community/docker-coq-action/pull/56#pullrequestreview-652829775)? Namely:

Sorry @Bruno-366 I've just realized you already committed this :)

Reply:
If I go to the actions tab and click on any "set up this workflow" I get sent to a page where I am creating a new yml file under `.github/workflows/`, only difference is that to the right there is a tab where you can search actions from the market place. Type in docker-coq and it says "Copy and paste the following snippet into your .yml file." 

![Screenshot 2021-05-07 at 12 54 58](https://user-images.githubusercontent.com/81762173/117439800-969d0d80-af33-11eb-83af-28d1cfe56ec6.png)
![Screenshot 2021-05-07 at 12 55 07](https://user-images.githubusercontent.com/81762173/117439797-969d0d80-af33-11eb-81e6-765d4db7e64e.png)
![Screenshot 2021-05-07 at 12 55 18](https://user-images.githubusercontent.com/81762173/117439796-96047700-af33-11eb-840d-009b7fbbcd1a.png)
![Screenshot 2021-05-07 at 12 55 26](https://user-images.githubusercontent.com/81762173/117439791-94d34a00-af33-11eb-8b61-d363786a7a98.png)

Reply:
But given that the `yml` in the marketplace isn't the same as `action.yml`, I'll follow your suggestion.